sw/device/lib/testing/rand_testutils.c

```diff
@@ -7,17 +7,37 @@
 #include <stdbool.h>
 #include <stdint.h>
 
+#include "sw/device/lib/arch/device.h"
+#include "sw/device/lib/dif/dif_rv_core_ibex.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/rv_core_ibex_testutils.h"
 #include "sw/device/lib/testing/test_framework/check.h"
 
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
 /**
  * The polynomial co-efficients used in the 32-bit LFSR implementation.
  *
  * This implementation matches the RTL design at `hw/ip/prim/rtl/prim_lfsr.sv`.
  */
-const uint32_t kLfsrCoefficients = 0x80000057;
+static const uint32_t kLfsrCoefficients = 0x80000057;
+
+/**
+ * Timeout in microseconds to fetch random data from entropy source.
+ */
+static const uint32_t kRandomDataFetchTimeoutMicros = 1000;
 
 uint32_t rand_testutils_gen32(void) {
-  static uint32_t lfsr = 1;
+  static uint32_t lfsr;
```

Comment:
```suggestion
  static uint32_t lfsr = 0;
```
Just to avoid reading from an uninitialized variable.

Reply:
Good catch, done!